extractor/filesystem/language/javascript/packagelockjson/extractor.go

```diff
@@ -158,7 +373,11 @@ func (e Extractor) ToPURL(i *extractor.Inventory) (*purl.PackageURL, error) {
 }
 
 // ToCPEs is not applicable as this extractor does not infer CPEs from the Inventory.
-func (e Extractor) ToCPEs(i *extractor.Inventory) ([]string, error) { return []string{}, nil }
+func (e Extractor) ToCPEs(_ *extractor.Inventory) ([]string, error) { return []string{}, nil }
+
+// Ecosystem returns the OSV ecosystem ('npm') of the software extracted by this extractor.
+func (e Extractor) Ecosystem(_ *extractor.Inventory) (string, error) {
+	return npmEcosystem, nil
```

Comment:
Inline npmEcosystem

Reply:
Removed/inlined.

Reply:
I just replaced the entire file with the osv-scanner version, which is why that's there. With #215 we can inline all these ecosystems consts now.